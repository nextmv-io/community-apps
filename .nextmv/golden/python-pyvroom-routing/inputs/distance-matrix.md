# distance-matrix

A test input with the distance matrix and speed given. The app will use them to
determine the travel cost between locations.
