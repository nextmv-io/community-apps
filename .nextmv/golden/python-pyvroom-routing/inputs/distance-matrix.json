{
  "defaults": {
    "vehicles": {
      "start_location": {
        "lon": 7.625,
        "lat": 51.9622
      },
      "end_location": {
        "lon": 7.625,
        "lat": 51.9622
      },
      "start_time": "2024-03-19T11:00:00+00:00",
      "capacity": 5
    },
    "stops": {
      "quantity": 1
    }
  },
  "vehicles": [
    {
      "id": "vehicle-1"
    },
    {
      "id": "vehicle-2"
    }
  ],
  "stops": [
    {
      "id": "location-1",
      "location": {
        "lon": 7.6228,
        "lat": 51.9624
      }
    },
    {
      "id": "location-2",
      "location": {
        "lon": 7.6353,
        "lat": 51.9584
      }
    },
    {
      "id": "location-3",
      "location": {
        "lon": 7.6233,
        "lat": 51.9638
      }
    },
    {
      "id": "location-4",
      "location": {
        "lon": 7.6163,
        "lat": 51.9635
      }
    },
    {
      "id": "location-5",
      "location": {
        "lon": 7.6045,
        "lat": 51.9509
      }
    },
    {
      "id": "location-6",
      "location": {
        "lon": 7.6292,
        "lat": 51.9577
      }
    },
    {
      "id": "location-7",
      "location": {
        "lon": 7.6385,
        "lat": 51.9694
      }
    },
    {
      "id": "location-8",
      "location": {
        "lon": 7.6628,
        "lat": 51.9616
      }
    },
    {
      "id": "location-9",
      "location": {
        "lon": 7.6217,
        "lat": 51.9667
      }
    },
    {
      "id": "location-10",
      "location": {
        "lon": 7.6299,
        "lat": 51.9582
      }
    }
  ],
  "distance_matrix": [
    [0, 1840, 583, 842, 2287, 994, 1942, 3549, 1016, 1070, 442, 442, 442, 442],
    [
      1954, 0, 1464, 2068, 2983, 806, 1409, 2530, 1759, 730, 1512, 1512, 1512,
      1512
    ],
    [583, 1734, 0, 742, 2432, 1139, 1746, 3443, 433, 1215, 291, 291, 291, 291],
    [838, 2356, 1099, 0, 2356, 1510, 2272, 4065, 726, 1586, 958, 958, 958, 958],
    [
      2632, 4040, 2893, 2154, 0, 3059, 4252, 5859, 2860, 3135, 2752, 2752, 2752,
      2752
    ],
    [1300, 1122, 1149, 1769, 2255, 0, 1662, 2880, 1564, 76, 858, 858, 858, 858],
    [
      2060, 1439, 1477, 2081, 3868, 1675, 0, 2380, 1526, 1599, 1768, 1768, 1768,
      1768
    ],
    [
      3747, 2530, 3164, 3768, 5169, 2992, 2392, 0, 3459, 2916, 3455, 3455, 3455,
      3455
    ],
    [
      1267, 1782, 1197, 712, 2911, 1939, 1657, 3491, 0, 2015, 1387, 1387, 1387,
      1387
    ],
    [1224, 1046, 1073, 1744, 2331, 76, 1586, 2804, 1488, 0, 782, 782, 782, 782],
    [442, 1443, 291, 962, 2291, 998, 1545, 3152, 724, 1074, 0, 0, 0, 0],
    [442, 1443, 291, 962, 2291, 998, 1545, 3152, 724, 1074, 0, 0, 0, 0],
    [442, 1443, 291, 962, 2291, 998, 1545, 3152, 724, 1074, 0, 0, 0, 0],
    [442, 1443, 291, 962, 2291, 998, 1545, 3152, 724, 1074, 0, 0, 0, 0]
  ]
}
